cou/steps/plan.py

```diff
@@ -596,7 +595,27 @@ async def _get_upgradable_hypervisors_machines(
     if cli_force:
         return nova_compute_machines
```

Comment:
Just for consistency, let's also log in this occasion and let the user know that all  hypervisors were selected. Something like

```python
logger.info("Selected all hypervisors: %s", all_hypervisors)
```